### ISSUE TITLE
ddev config --web-environment overwrites all values

### ISSUE THREAD

**Issue by justafish** [drud/ddev#3691]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Running `ddev config --web-environment="FOO=bar"` overwrites all existing values

### Expected Behavior

Running `ddev config --web-environment="FOO=bar"` should append the variable to an existing list of variables, not overwrite everything

### Steps To Reproduce

```
$ composer create-project drupal/recommended-project ddev-test
$ cd ddev-test
$ ddev config
$ ddev config --web-environment="FOO=bar"
$ cat .ddev/config.yaml 
name: ddev-test
type: drupal9
docroot: web
php_version: "8.0"
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
database:
  type: mariadb
  version: "10.3"
nfs_mount_enabled: false
mutagen_enabled: false
use_dns_when_possible: true
composer_version: "2"
web_environment:
- FOO=bar
.
.
$ ddev config --web-environment="BAR=baz"
$ cat .ddev/config.yaml
name: ddev-test
type: drupal9
docroot: web
php_version: "8.0"
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
database:
  type: mariadb
  version: "10.3"
nfs_mount_enabled: false
mutagen_enabled: false
use_dns_when_possible: true
composer_version: "2"
web_environment:
- BAR=baz
nodejs_version: "16"
.
.
```

**Comment by rfay** [comment]
Yes, it always has, because you're setting the `web_environment` value. I certainly think it would be valuable to have a `--add-to-web-environment` or something... 

Related:
* https://github.com/drud/ddev/issues/3598

**Comment by justafish** [comment]
@rfay it was the output of `ddev config --help` that led me to think that -

```
--web-environment string           Add environment variables to web container: --web-environment="TYPO3_CONTEXT=Development,SOMEENV=someval"
```

So I was expecting it to just add variables, not "add and also remove" 😊

**Comment by rfay** [comment]
Interesting, that flag description was written when there wasn't an easy way to add environment variables to the web container (you had to do a `docker-compose.env.yaml` to do it. So it meant "add environment variables that you weren't able to add before." :) 

I guess now that we're farther down the line it should have better wording. What would you suggest? "Set environment variables in web container" ? PR is welcome, but just the thinking is good.

**Comment by chrisfromredfin** [comment]
Just wordsmithing here but I think "Set the environment variables in web container" - I think adding the definite article clarifies that you're essentially setting all of them.

**Comment by rfay** [comment]
Would love your review of
* https://github.com/drud/ddev/pull/3692

**Comment by chrisfromredfin** [comment]
Difference in language between config.go and config-global.go (one says the twice, other only once). Make consistent.

In one spot on line 62 you have DDEV_provided instead of DDEV-provided
